LAKMobile/src/screens/AddJob.tsx

```diff
@@ -593,7 +600,8 @@ export function AddJob({ navigation, route }: AddJobProps) {
         />
         {formType === "edit" && (
           <AppButton
-            onPress={handleDeleteJob}
+            onPress={() => setConfirmationVisible(true)}
+            //onPress = {handleDeleteJob}
```

Comment:
I think we can also remove this here since we're moving to the confirmation module